src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is starting its work.");
```

Comment:
The message should be more informative and indicate that it is an `Excavator` that is working. Consider including the class name in the message.